packages/light-client/README.md

```diff
@@ -48,7 +48,7 @@ lodestar lightclient \
 
 ## Light-Client Programmatic Example
 
-For this example we will assume there is a running beacon node at `https://beacon-node.your-domain.com`
+For this example we will assume there is a running beacon node at `https://lodestar-mainnet.chainsafe.io`
```

Comment:
Are we sure we want to broadcast this URL?  @philknows @wemeetagain

Reply:
If anyone have different opinion we can revert it in other PR.